src/js/components/LoginPage/Login.scss

```diff
@@ -0,0 +1,127 @@
+$primary-color: #fff;
+
+.login{
+  background-color: #bbb;
+  min-height: 100vh;
+  color: $primary-color;
+  padding-bottom: 30px;
+  & span {
+    display: block;
+    overflow: hidden;
+    padding: 0 4px 0 6px
+  }
+}
+
+label {
+  float: left
+}
+
+input {
+  width: 100%
+}
+.form-input{
+  background-color: transparent;
+  border: 1px solid $primary-color;
+  font-size: 15px;
+  height: 35px;
+  padding-left: 10px;
+  color: $primary-color;
+}
+
+.margin-top-10{
+  margin-top: 10px;
+}.margin-top-20{
+  margin-top: 20px;
+}
+.margin-left-20{
+  margin-left: 20px;
+}
+
+.margin-top-30{
+  margin-top: 30px;
+}
+
+.btn-login-options{
+  border-radius: 100%;
+  border: 1px solid grey;
+  margin-right: 8px;
+  width: 30px;
+  height: 30px;
+  &.facebook{
+    background-color: $primary-color;
+  }
+  &.google{
+    background-color: $primary-color;
+  }
+  &.twitter{
+    background-color: $primary-color;
+  }
+}
+
+.div-membershiptype{
+  border: 1px solid $primary-color;
+  min-height: 100px;
+  margin-bottom: 10px;
+  padding-top: 10px;
+}
+
+.no-background{
+  background-color: transparent !important; // Had to overwrite bootstrap default
+}
+
+.nav-link{
+  padding-left: 0px ;
+  font-size: 9px;
+  color: $primary-color;
+}
+
+.btn-login{
+  color: $primary-color;
+  background-color: #333;
+  height: 40px;
+  width: 120px;
+  border-radius: 20px;
+  float: left;
+  font-size: 14px;
+  border: 0px;
+}
+
+.btn-signup{
+  color: $primary-color;
+  background-color: #333;
+  height: 40px;
+  width: 120px;
+  border-top-left-radius: 20px;
+  border-bottom-left-radius: 20px;
+  float: right;
+
+  font-size: 14px;
+  border: 0px;
+  padding-right: 0px;
+
+
+}
+
+.div-btn{
+  width: 100%;
+  padding-right: 0px;
+  // !important;
+}
+
+.text-forgotpassword{
+  font-size: 11px;
+  float: left;
+  &:hover{
+    cursor: pointer;
+  }
+}
+
+.display-flow-root{
+  // Had to use this to override bootstraps row property
+  display: flow-root !important;
+  margin-left: 20px !important;
+}
+
+.border-danger{
+  //border: 1px solid red;
```

Comment:
GC and add empty line at the end of the file

Reply:
Removed GC